components/generic/Select/Select.tsx

```diff
@@ -0,0 +1,35 @@
+import { UnfoldMoreIcon } from '../Icons/UnfoldMoreIcon';
+
+type Option = {
+  name: string;
+  value: string;
+};
+
+type SelectProps = {
+  name: string;
+  label: string;
+  options: Option[];
+};
+
+export const Select = ({ name, label, options }: SelectProps) => {
+  return (
+    <label className="flex flex-col">
+      <span className="text-xs leading-4 font-normal text-gray-800 mb-1">{label}</span>
+      <div className="relative">
+        <select
+          className="w-full px-2 py-2 bg-white border rounded border-gray-200 placeholder:text-gray-600 text-gray-700 text-xs leading-4 font-normal focus:outline-none appearance-none"
+          name={name}
+        >
+          {options.map((opt) => (
+            <option key={opt.name} value={opt.value}>
+              {opt.value}
+            </option>
+          ))}
+        </select>
+        <div className="absolute right-2 bottom-1.5">
+          <UnfoldMoreIcon />
+        </div>
```

Comment:
Maybe there's a tailwind class for centering inside relative container? Why don't you pass `className` prop to svg?